time node_timestamp
string node_label
